examples/hydrogen-2/vercel.json

```diff
@@ -0,0 +1,6 @@
+{
+  "env": {
+    "SESSION_SECRET": "foobar",
```

Comment:
Should these be in the `vercel.json`?

Reply:
See my note in the PR description. Ideally we'd prompt for the user to enter these when importing the template on vercel.com/new, but I don't think we currently have a mechanism to do so from within these framework templates.